examples/dataset_exploration/matpes/eda.py

```diff
@@ -26,6 +26,7 @@
 
 
 # %%
+# DEBUG: following data doesn't exist
 with gzip.open(f"{module_dir}/20240214-MatPES-178070-r2SCAN.json.gz", "rt") as file:
```

Comment:
Not sure where to download this dataset?

Reply:
yes, that dataset is not public yet. i should have added a comment about that. [just did so now](https://github.com/janosh/pymatviz/commit/b6cdca34a3643aa9dc04070bb92470bb20e7ad3e#diff-2490fe956ff43be755c667a04bdfee7e0d1d6a13d8c92a62175d5ab33e94d826R28-R30)

Reply:
Thanks for the input!